### DISCUSSION TITLE
Improve the information recognition of nodata alarms

### DISCUSSION THREAD

**Discussion by lucklymore** [grafana/grafana#54852] [Alerting]
### **Request：Improve the information recognition of nodata alarms**
in default nodata alarm message  only print "no data \ refid \  datasourceid",
It lacks a clear identification to distinguish the content of the current alert,The current nodata alarm information is very limited in judging the content of the alarm.
It is recommended that the alarm message can feed back the actual query statement and the alarm condition to identify the actual content of the alarm , especially in multiple compound query statements and alarm conditions.

Thanks,

![image](https://user-images.githubusercontent.com/31549331/188896482-50f6b90e-60c5-4257-aa23-3be3c91ce71f.png)

![image](https://user-images.githubusercontent.com/31549331/188896737-cf908e8a-8427-4563-8302-07cedbf28225.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
